overlord/snapstate/snapmgr_test.go

```diff
@@ -3296,8 +3630,59 @@ func (s *snapmgrTestSuite) TestUpdateCanDoBackwardsNotQuiteYet(c *C) {
 		Current:  si11.Revision,
 	})
 
-	_, err := snapstate.Update(s.state, "some-snap", "channel-for-7", s.user.ID, 0)
-	c.Assert(err, ErrorMatches, `revision 7 of snap "some-snap" already installed`)
+	chg := s.state.NewChange("refresh", "refresh a snap")
```

Comment:
Let me start with: yay! We also want to update the test name I think `TestUpdateCanDoBackwardsNotQuiteYet`->`TestUpdateCanDoBackwardsHellYeah` maybe?